docs/installing.md

```diff
@@ -129,3 +124,74 @@ It is also recommended to have:
 
 Additional resources may be required depending on the workloads you plan to run.
 
+### Installation via .deb Package
+
+Add the Rancher Desktop repository and install Rancher Desktop with:
+
+```
+curl https://download.opensuse.org/repositories/isv:/Rancher:/stable/deb/Release.key | sudo apt-key add -
```

Comment:
Getting a `sudo` prompt in the middle of `curl` output slightly misleading. Maybe `curl -s` would help, but I also prefer pre-warming `sudo`, like:
```
sudo echo ok
curl -s https://download.opensuse.org/repositories/isv:/Rancher:/stable/deb/Release.key | sudo apt-key add -
...
```

There might be more conventional ways of doing this.